include/net/nrf_cloud.h

```diff
@@ -994,6 +1016,10 @@ bool nrf_cloud_fota_is_type_enabled(const enum nrf_cloud_fota_type type);
  */
 int nrf_cloud_fota_job_start(void);
 
+int nrf_cloud_fota_support_init(struct nrf_cloud_fota_support_ctx *ctx);
```

Comment:
Since there will only ever be a single `nrf_cloud_fota_support` instance, maybe it would make sense to have `nrf_cloud_fota_support_init` copy the `ctx` struct to a private global context struct, and have `nrf_cloud_fota_support_start` and `nrf_cloud_fota_support_process` be parameter-less.

If you move in that direction, consider also renaming to `cfg` instead of `ctx`, and having it be pass by value rather than pass by pointer

Reply:
I modeled this after what Justin did in the nrf_cloud_rest library, which accepts `struct nrf_cloud_fota_poll_ctx *` to be passed to every API.  Since this library uses that library (when built for REST), I'd like to leave it this way.